util/cprofiler_flamegraph.md

```diff
@@ -0,0 +1,17 @@
+## Requirements
+
+CProfilerFlameGraph depends on snakeviz to render flamegraph as html file. Main snakeviz repo requires renders flamegraph by launching webserver and rendering html through that. On some enviornments you want to just get a statically renderable html that can be shared. Snakeviz's author provides utility for this at https://gist.github.com/jiffyclub/6b5e0f0f05ab487ff607. This util just combined the patch from gist and added a util that does not need to launch server.
+
+## Installation
+
+pip install snakeviz
```

Comment:
Create an install_requirements.sh for this?

Reply:
It is just snakeviz pip install, so didnt quite do it.